docs/user/get-started.md

```diff
@@ -0,0 +1,339 @@
+
+## About
+
+Vald is a distributed highly scalable and fast approximate nearest neighbor dense feature vector search engine.<br>
+Vald can handle large amount of specific objects, such as image, text, video, and etc.<br>
+Vald is designed base on Cloud Native.
+It uses the fastest ANN Algorithm [NGT](https://github.com/yahoojapan/NGT) to search neighbors.
+(If you are interested in ANN benchmarks, please refer to [the official website](http://ann-benchmarks.com/).)
+
+### Main Features
+
+- Auto Indexing
+    - Normally, when changing the Graph Index, the Graph must be locked, but Vald uses distributed index graph, it is extremely difficult for the user to reconstruct the Graph.
+    - Therefore, Vald automatically indexes distributed Graphs sequentially.
+
+- Ingress/Egress Filltering
+    - Vald has it's own highly customizable Ingress/Egress filter.
+    - Which can be configured by the user to fit the gRPC interface.
+        - Ingress Filter: Ability to Vectorize through filter on request.
+        - Egress Filter: A function to rerank or filter the Search response with your own algorithm.
+
+- Horizontal Scaling
+    - Vald is a cloud-native vector search engine running on Kubernetes, which enables horizontal scalling of memory and cpu for billion scale of vector data.
+
+- Auto Index Backup
+    - Vald has auto index backup feature using MySQL + Redis or Cassndora which enables disaster recovery.
+
+- Distributed Indexing
+    - Vald distribute indexes vecotr to multiple agent. Which means each agent has different graph index.
+
+- Index Replication
+    - Vald stores each index in multiple agents which enables index replicas.
+    - Rebalance the replica when some pods go down.
+
+- Easy to use
+    - Vald can be easily installed in a few steps and Vald is highly customizable.
+
+### Requirements
+
+- k8s:  v1.17 ~
+- go:   v1.14 ~
+- helm: v3 ~
+- libhdf5 (_`libhdf5` is only required doing Get-Started, is not required Vald._)
+
+If helm/hdf5 is not installed, please install helm (see below details or [here](https://helm.sh/docs/intro/install))/hdf5 (see below details or [here](https://www.hdfgroup.org/)).
+
+<details><summary>optional installation</summary><br>
+install helm
+
+```bash
+curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
+```
+install hdf5
+```bash
+## yum
+yum install -y hdf5-devel
+
+## apt
+apt-get install libhdf5-serial-dev
+
+## homebrew
+brew install hdf5
+```
+</details>
+
+## Starting Vald on k8s cluster
+
+### Deploy
+
+This section shows how to deploy Vald with Scylla, which is used as a datastore for index-metadata and backup-manager.
+If you want to learn about Scylla, please refer to [the official website](https://www.scylladb.com/).
+
+1. Confirm the right cluster to deploy
+
+    ```bash
+    kubectl cluster-info
+    ```
+
+2. Prepare Scylla DB and k8s metrics-server
+
+    At first, you have to apply scylla as backup database.
+
+    ```bash
+    cd path/to/vald/root 
```

Comment:
can delete this line?

Reply:
maybe it's ok we don't need delete this section

Reply:
we can delete :)